src/components/Text/index.tsx

```diff
@@ -45,13 +46,13 @@ const generateStyles = ({
       ? capitalize(PROMINENCES.default)
       : capitalize(PROMINENCES[prominence])
 
-  const themeColor = theme.colors[color]
+  const themeColor = theme.colors[color as Color]
   const text = `text${definedProminence}${
     disabled ? 'Disabled' : ''
   }` as keyof typeof themeColor
 
   return `
-    color: ${theme.colors[color][text]};
+    color: ${color === 'inherit' ? color : theme.colors[color][text]};
```

Comment:
We could have 
```suggestion
    color: ${color ? theme.colors[color][text] : 'inherit'};
```

And avoid everything else :)

Reply:
@chambo-e If I do that types won't show that inherit is the default value for color

Reply:
We don't need to, that's already how css works :)